nilearn/surface/surface.py

```diff
@@ -1423,6 +1246,39 @@ def __init__(self, coordinates, faces):
         self.faces = faces
         self.n_vertices = coordinates.shape[0]
 
+    def __getitem__(self, index):
```

Comment:
We can allow `InMemoryMesh` to be subscriptable with [0] and [1] and throw a future deprecation warning like this

Reply:
Or we could just support this subscriptability forever...